### ISSUE TITLE
Script loops when it can't get to 

### ISSUE THREAD

**Issue by faceboogle** [soolar/sl_ascend#244]
2CRS HC run, with Canadia moon-sign
`Not eating or drinking anything, since we don't know what's good...
Looking at the billiards room: 14 <= 10 <= 18
Not quite boozed up for the billiards room... we'll be back.
At the shore, la de da!

Visit to Beach: The Shore, Inc. Travel Agency in progress...
You can't get to that area.

Visit to Beach: The Shore, Inc. Travel Agency in progress...
You can't get to that area.

Visit to Beach: The Shore, Inc. Travel Agency in progress...
You can't get to that area.`

Perhaps 2CRS has broken the error handling that would have escaped from this loop?

**Comment by soolar** [comment]
Is this still happening for you? I haven't encountered it myself.

**Comment by faceboogle** [comment]
Yes, had it again today. It couldn't get to the Desert Beach. Since this was a gnoll sign ascension I easily bought the parts for a meat car manually, ran again and it was fine (HC 2CRS if that matters)
